### DISCUSSION TITLE
Is your app being copied?

### DISCUSSION THREAD

**Discussion by KadokawaJP** [mrkai77/Loop#332] [Q&A]
### Feature Request Description

I couldn't find an appropriate emoji to put in the title, though it doesn't really matter.
I found an app in the App Store: https://apps.apple.com/app/align-organize-app-windows/id6480428845 , which looks like it copied your app. btw, it's getting downloads through free in-app purchases for a limited time.

### Screenshots

<img width="812" alt="01" src="https://github.com/MrKai77/Loop/assets/96008024/bbf2a1a9-35a0-4e51-bbfb-78f9b198464a">

### Final Checks

- [X] My issue title is descriptive
- [X] This is a single feature request (multiple feature requests should be reported individually)

**Comment by mrkai77** [comment]
Thank you very much for bringing this to my attention!

I do agree that this app looks too similar to Loop, and definitely looks like a copy. I have just submitted a claim [here](https://www.apple.com/legal/intellectual-property/dispute-forms/app-store/app-store-dispute.html), so hopefully it gets dealt with!

**Comment by mrkai77** [comment]
Alright, as of their latest version (2.3.0), they have removed the Loop-styled radial menu :)

**Comment by LiXin-x** [comment]
Align's bugs are just like Loop's.
It's simply amazing!!!
It's a homage to Loop from the bottom of the code.
